packages/devtools_app/lib/src/shared/new_issue.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../devtools.dart' as devtools;
+import 'globals.dart';
+import 'utils.dart';
+
+const newDevToolsIssueUriDisplay = 'github.com/flutter/devtools/issues/new';
+
+Uri newDevToolsIssueUri({String? issueDetails}) {
+  final issueBodyItems = issueLinkDetails();
+  if (issueDetails != null) issueBodyItems.insert(0, issueDetails);
+  final issueBody = issueBodyItems.join('\n');
+
+  return Uri.parse('https://$newDevToolsIssueUriDisplay').replace(
```

Comment:
It is possible for the url param to have too many characters. Should we try to limit the length of the body details in order to avoid hitting that?
https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue

Reply:
Added limit. Thanks.